tree/ntuple/v7/inc/ROOT/RNTupleProcessor.hxx

```diff
@@ -39,9 +39,11 @@ struct RNTupleProcessorEntryLoader;
 /// Used to specify the underlying RNTuples in RNTupleProcessor
 struct RNTupleOpenSpec {
    std::string fNTupleName;
-   std::string fStorage;
+   std::variant<std::string, TDirectory *> fStorage;
```

Comment:
I think we need to discuss a bit the public interface of this. I am not sure it's going to be easily usable from Python, which might or might not be what we want.